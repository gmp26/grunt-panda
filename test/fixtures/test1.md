
# title {.reveal}

## subtitle

<%= panda.test1.files["test/actual/test1.html"] %>

_emphasis_
__bold__
