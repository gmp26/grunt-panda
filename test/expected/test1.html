<section id="title" class="level1 reveal">
<h1>title</h1>
<section id="subtitle" class="level2">
<h2>subtitle</h2>
<p>test/fixtures/test1.md</p>
<p><em>emphasis</em> <strong>bold</strong></p>
</section>
</section>
